contracts/UFragmentsPolicy.sol

```diff
@@ -182,6 +182,31 @@ contract UFragmentsPolicy is Ownable {
         rebaseLag = rebaseLag_;
     }
 
+    /**
+     * @notice Sets the hyper-parameters which control the timing and frequency of
+     *         rebase operations.
+     *         a) the minimum time period that must elapse between rebase cycles.
+     *         b) the rebase window offset parameter.
+     *         c) the rebase window length parameter.
+     * @param minRebaseTimeIntervalSec_ More than this much time must pass between rebase
```

Comment:
```suggestion
     * @param minRebaseTimeIntervalSec_ At least this much time must pass between rebase
```

Reply:
👍

Reply:
Revering back to `More than` @ahnaguib 	@brandoniles